internal/bpf/bpfWrapper.c

```diff
@@ -212,7 +211,7 @@ int Load_attach_bpf_xdp_pass(char *ifname) {
 	}
 
 	/* Attach the program to the interface at the xdp hook */
-	err = xdp_program__attach(prog, ifindex, XDP_FLAGS_UPDATE_IF_NOEXIST, 0);

```

Comment:
Move to xdp loader also

Reply:
I could also remove the bpf code from the CNI altogether by leveraging the DPSyncer and doing everything in the DP as part of this PR?